pkg/kube/constants.go

```diff
@@ -76,6 +76,9 @@ const (
 	// ConfigMapJenkinsDockerRegistry is the ConfigMap containing the Docker Registry configuration
 	ConfigMapJenkinsDockerRegistry = "jenkins-x-docker-registry"
 
+	// ConfigMapNameJXInstallConfig is the ConfigMap containing the jx installation's CA and server url. Used by jx login
+	ConfigMapNameJXInstallConfig = "jx-install-config"
```

Comment:
I think it's better to have it called "jx-install-config" or something similar because cluster-config kinda suggests that there's only one install of jx on a cluster and we might have many installations. If it's a real pain I can just change the string for now and we can revisit.

Reply:
It's ok on my side. Please could you send a PR to update the cdx backend?

Reply:
Sure thing.